websites/S/Spyke/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "726466786916171796",
+		"name": "iamhv"
+	},
+	"service": "Spyke",
+	"description": {
+		"en": "Spyke - Community Discussion"
+	},
+	"url": "spyke.social",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/v10pkZA.png",
+	"thumbnail": "https://i.imgur.com/55dCzB1.png",
```

Comment:
This thumbnail is very small. Replace it with a larger promotional image or screenshot.